library/systemd/src/lib/yast2/systemd_unit.rb

```diff
@@ -3,6 +3,14 @@
 require "ostruct"
 
 module Yast
+  # Represent a fail when systemctl command tries to refresh the service
+  class CouldNotRefreshUnitError < StandardError
```

Comment:
just thinking about it, why not having some reasonable ancestor for all that systemd errors?

Reply:
or service errors?

Reply:
Make sense to me, as long as we decide to cover more scenarios adding more specific exceptions.

Reply:
Yes, I agree with @jreidinger. I would create a new exception based on existing `SystemctlError`, for example:

```
class SystemctlRefreshError < SystemctlError; end
```

Using a subclass we ensure that all code catching current `SystemctlError` will continue catching the new exception `SystemctlRefreshError`.